File: app/src/main/java/com/alesno/testtaskispring/model/objectbox/entity/VideoObj.java
package com.alesno.testtaskispring.model.objectbox.entity;

import io.objectbox.annotation.Backlink;
import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import io.objectbox.annotation.NameInDb;
import io.objectbox.relation.ToMany;
import io.objectbox.relation.ToOne;

@Entity
public class VideoObj {

    @Id
    public long id;

    @Backlink(to = "video")
    public ToMany<ExpertObj> experts;

    @NameInDb("id_response")
    public String idResponse;

    //@Backlink(to = "video")
    public ToOne<MetaObj> meta;

    @NameInDb("favorite")
    public Boolean isFavorite;

    @NameInDb("progress_time")
    public Long progressTime;
}

File: app/src/main/java/com/alesno/testtaskispring/model/objectbox/VideoObj.kt
package com.alesno.testtaskispring.model.objectbox

data class VideoObj(
    val experts: List<ExpertObj>,
    val idResponse: String,
    val meta: MetaObj,
    var id: Long = 0,
    var isFavorite: Boolean = false,
    var progressTime: Long = 0
)